Userland/Shell/Builtin.cpp

```diff
@@ -58,10 +85,10 @@ int Shell::builtin_where(int argc, char const** argv)
             }
         }
 
-        auto path = Core::File::resolve_executable_from_environment(current_command);
-        if (path.has_value()) {
+        auto const& executables = find_matching_executables_in_path(current_command);
```

Comment:
Drop the `&` here, temporary lifetime extension isn't needed or wanted.

Reply:
Thanks for the feedback. Done.